test/functional/feature_llmq_is_retroactive.py

```diff
@@ -24,6 +24,17 @@ def set_test_params(self):
         # -whitelist is needed to avoid the trickling logic on node0
         self.set_dash_test_params(5, 4, [["-whitelist=127.0.0.1"], [], [], [], ["-minrelaytxfee=0.001"]])
 
+    def check_no_is(self, txid, node):
+        try:
+            self.log.info(f"Expecting no InstantLock for {txid}")
+            assert not node.getrawtransaction(txid, True)["instantlock"]
+        except:
+            assert False
```

Comment:
can't this be simply
```suggestion
        self.log.info(f"Expecting no InstantLock for {txid}")
        assert not node.getrawtransaction(txid, True)["instantlock"]
```
?